common/api/imodeljs-backend.api.md

```diff
@@ -232,7 +231,7 @@ export class AnnotationElement2d extends GraphicalElement2d {
     static get className(): string;
 }
 
-// @beta
+// @beta @deprecated

```

Comment:
You've deprecated a bunch of APIs without suggesting replacements - are replacements available for any of them?
Are we keeping the `@beta` ones around because we or apps are using them and we don't want to have to update all that code right now?

Reply:
I deprecated the whole "auto push" api - it was never a good idea.

All of the @beta api's related to that have this message:
```ts
 * @deprecated auto-push is no longer supported and will be removed in 3.0.
```
Frankly, I don't know if anyone is using it - I hope not.

Is that what you mean?

Reply:
AutoPush was invented for the Plant application.

Reply:
> All of the @beta api's related to that have this message:

Thanks - I was looking at the .api.md files, which I now realize omit those details.

Reply:
> AutoPush was invented for the Plant application.

That's why I deprecated it instead of removing it. But, they need to rework their edit-from-web strategy to use IPC. Then, if they want an "auto-push" concept (I hope they won't), they can move all this code into their application. It shouldn't be part of our api.